### PR TITLE
[Web-28] search form

### PR DESCRIPTION
**Pull request by emilyPhee**
- Implement `SearchControl` component and add in `CourseSearchWindow`
- Change dropdown and input border color for light theme

### CODE REVIEW

**Review by emilyPhee — COMMENTED** [review]
reviewed

File: new-nextjs-client/src/components/index-page/CourseSearchWindow/SearchControl/SearchControl.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        <FormLabel
+          fontSize={fontSizeMD}
+          letterSpacing={letterSpacingMD}
+          color={theme === 'light' ? defaultText : defaultTextDark}
+        >
+          Number
+        </FormLabel>
+        <AppInput
+          onChange={(e) => {
+            if (e.target.value === '') {
```
Comment (anthopark) [inline new-nextjs-client/src/components/index-page/CourseSearchWindow/SearchControl/SearchControl.tsx]: let's change this line to `setInputValue(e.target.value === '' ? null : e.target.value)`

**Review by anthopark — APPROVED** [review]
Looks good!

**Comment by emilyPhee** [comment]
[[Web] Search form in CourseSearchWindow Component](https://trello.com/c/53nNfQ9d/28-web-search-form-in-coursesearchwindow-component)

**Pull request merged**